contrib/tests/actions/chains/test_quickstart_rules.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 vars:
-        tested_rule: examples.webhook_file
+        tested_rule: examples.examples.webhook_file
```

Comment:
This is slightly confusing now.

It might be better to rename the rule and remove `examples.` prefix from the rule name.